source/sass/views/fellowships/shared.scss

```diff
@@ -46,6 +46,12 @@
   }
 }
 
+[id^="view-fellows-directory"] {
```

Comment:
so it includes id=

- `view-fellows-directory`
- `view-fellows-directory-<program>` like `view-fellows-directory-science` and `view-fellows-directory-media` etc